index8x8.html

```diff
@@ -38,90 +38,88 @@ <h1 class="title">2048</h1>
+        <div id="column5" class="grid-box col-xs-2" myAttr='canMove' combine='canCombine'>0</div>
+        <div id="column6" class="grid-box col-xs-2" myAttr='canMove' combine='canCombine'>0</div>
+        <div id="column7" class="grid-box col-xs-2" myAttr='canMove' combine='canCombine'>0</div>
+        <div id="column8" class="grid-box col-xs-2" myAttr='canMove' combine='canCombine'>0</div>
+      </div>
+      <div class = "row" id="grid_6">
+        <div id="column1" class="grid-box col-xs-2" myAttr='canMove' combine='canCombine'>0</div>
+        <div id="column2" class="grid-box col-xs-2" myAttr='canMove' combine='canCombine'>0</div>
+        <div id="column3" class="grid-box col-xs-2" myAttr='canMove' combine='canCombine'>0</div>
+        <div id="column4" class="grid-box col-xs-2" myAttr='canMove' combine='canCombine'>0</div>
+        <div id="column5" class="grid-box col-xs-2" myAttr='canMove' combine='canCombine'>0</div>
+        <div id="column6" class="grid-box col-xs-2" myAttr='canMove' combine='canCombine'>0</div>
+        <div id="column7" class="grid-box col-xs-2" myAttr='canMove' combine='canCombine'>0</div>
+        <div id="column8" class="grid-box col-xs-2" myAttr='canMove' combine='canCombine'>0</div>
+      </div>
+      <div class = "row" id="grid_7">
+        <div id="column1" class="grid-box col-xs-2" myAttr='canMove' combine='canCombine'>0</div>
+        <div id="column2" class="grid-box col-xs-2" myAttr='canMove' combine='canCombine'>0</div>
+        <div id="column3" class="grid-box col-xs-2" myAttr='canMove' combine='canCombine'>0</div>
+        <div id="column4" class="grid-box col-xs-2" myAttr='canMove' combine='canCombine'>0</div>
+        <div id="column5" class="grid-box col-xs-2" myAttr='canMove' combine='canCombine'>0</div>
+        <div id="column6" class="grid-box col-xs-2" myAttr='canMove' combine='canCombine'>0</div>
+        <div id="column7" class="grid-box col-xs-2" myAttr='canMove' combine='canCombine'>0</div>
+        <div id="column8" class="grid-box col-xs-2" myAttr='canMove' combine='canCombine'>0</div>
+      </div>
+      <div class = "row" id="grid_8">
+        <div id="column1" class="grid-box col-xs-2" myAttr='canMove' combine='canCombine'>0</div>
+        <div id="column2" class="grid-box col-xs-2" myAttr='canMove' combine='canCombine'>0</div>
+        <div id="column3" class="grid-box col-xs-2" myAttr='canMove' combine='canCombine'>0</div>
+        <div id="column4" class="grid-box col-xs-2" myAttr='canMove' combine='canCombine'>0</div>
+        <div id="column5" class="grid-box col-xs-2" myAttr='canMove' combine='canCombine'>0</div>
+        <div id="column6" class="grid-box col-xs-2" myAttr='canMove' combine='canCombine'>0</div>
+        <div id="column7" class="grid-box col-xs-2" myAttr='canMove' combine='canCombine'>0</div>
+        <div id="column8" class="grid-box col-xs-2" myAttr='canMove' combine='canCombine'>0</div>
+      </div>  
+    </section>
```

Comment:
Instead of hard coding these grids, you need to dynamically generate them using jQuery